extensions/wikia/Flags/FlagsExtractor.class.php

```diff
@@ -553,7 +553,7 @@ private function getTemplateParam( $isParamWithName, $paramsCounter, $paramOffse
 	 * @param string $message text
 	 * @param array $params additional parameters
 	 */
-	private function logInfoMessage( $message, $params = [] ) {
+	protected function logInfoMessage( $message, $params = [] ) {
```

Comment:
I think it's better not to be changing the visibility of a private method just to mock it as this is an implementation detail. I'd just mock `info` from the Loggable trait which would be used even if you changed this method and is public and leave this private.